### DISCUSSION TITLE
Example as a guide

### DISCUSSION THREAD

**Discussion by AZFARHAD24511** [USEPA/stateior#37] [Ideas]
The effort done in this field is really appreciated. Only one thing may be necessary to mention here is a simple example of how to use these commands and different codes, which is really vital, especially for those who are beginners. Is it possible that someone can provide an example of implementation of supply and consumption tables, supply and consumption balancing, converting it to IOT based on different assumptions? Especially the part about JGRAS algorithm

**Comment by AZFARHAD24511** [comment]
actually my supply and use table is not balance and i want to use GRAS but in the implementation of them i have propblem.does anyone has an example in this case?

**Comment by bl-young** [comment]
I'll point you to my answer in #39, where the wiki describes a sequence of model build.
